gis4wrf/core/transforms/wps_namelist_to_project.py

```diff
@@ -48,13 +48,20 @@ def convert_nml_to_project_domains(nml: dict) -> List[dict]:
 
     # Create CRS object from projection metadata.
     if map_proj == 'lat-lon':
-        if standlon != 0.0:
+        if stand_lon != 0.0:
             raise UnsupportedError('Rotated lat-lon projection is not supported')
         crs = CRS.create_lonlat()
     elif map_proj == 'lambert':
-        # It doesn't matter what the origin is. See wps_binary_to_gdal.py for details.
-        origin = LonLat(lon=standlon, lat=(truelat1 + truelat2)/2)
+        # It doesn't matter what the latitude origin is.
+        # Longitude is standard longitude and matters.
```

Comment:
? Rephrase.

Reply:
I re-wrote the existing text in wps_binary_to_gdal.py and then referred to that from here. It should hopefully now be clear for future reference.

Reply:
Agree. Also the use of `arbitrary_longitude_origin = 0`  is more informative!